modules/authorization/role-assignment/main.json

```diff
@@ -5,7 +5,7 @@
     "_generator": {
```

Comment:
@AlexanderSehr probably not related, but since this PR introduces main.json for the module. I can see the static tests are passing but we are getting another error as

```powershell

 Status Message: The request was incorrectly formatted. (Code:BadRequestFormat)
  
  Status Message: At least one resource deployment operation failed. Please list deployment operations for details. Please see https://aka.ms/arm-deployment-operations for usage details. (Code: DeploymentFailed)
   - The request was incorrectly formatted. (Code:BadRequestFormat)
  
  
  CorrelationId: 6fe59a5c-1a4f-4a57-9c2c-e53bd48e2bcd

```
I have seen this before but cannot remember when

[![Authorization - RoleAssignments](https://github.com/Azure/ResourceModules/actions/workflows/ms.authorization.roleassignments.yml/badge.svg?branch=users%2Falsehr%2F4157_avm_tags_01)](https://github.com/Azure/ResourceModules/actions/workflows/ms.authorization.roleassignments.yml)

Reply:
I've seen this before. The Authorization namespace showed up as a phantom from time to time. Will undo

Reply:
Removed